server-spi/src/main/java/org/keycloak/models/KeycloakSessionTask.java

```diff
@@ -25,5 +25,9 @@
 public interface KeycloakSessionTask {
 
     void run(KeycloakSession session);
+    
+    default boolean useExistingSession() {
```

Comment:
As this is public SPI interface, it'd be good to have some description of the method and when it's useful to use it, right?

Reply:
Can be done in a follow-up as we already have the required approvals.

Reply:
> As this is public SPI interface, it'd be good to have some description of the method and when it's useful to use it, right?

This is in a public module, but it's only usage is against the private spi logic - it (and related interfaces) could just as easily be moved to the private module.

Another approach would be to instead add a thread local toggled via a method like KeycloakModelUtils.setUseCurrentSessionForTransaction - but I didn't like the thought of adding another threadlocal when there was an existing hook to localize these changes to import export.

Reply:
> This is in a public module, but it's only usage is against the private spi logic - it (and related interfaces) could just as easily be moved to the private module.

+1 You're right. I think it'd need more polishing of public/private SPI modules. 

As users are not able to extend functionality about the KeycloakSessionTask in a supported way (needed in the private SPI module), the comment on the method is not needed - ignore my first suggestion.